tests/mmuconfig_input/mmuconfig_test_vectors.py

```diff
@@ -83,6 +83,36 @@
 
 """
 
+show_mmu_config_one = """\
+Pool: ingress_lossy_pool
+----  -------
+mode  dynamic
+type  ingress
+----  -------
+
+Pool: ingress_lossless_pool_hbm
+----  ---------
```

Comment:
There is no hbm pool in voq chassis. Can you please have example showing something like this ?

![image](https://github.com/sonic-net/sonic-utilities/assets/46945843/3855845f-8beb-41b8-b1a8-dd387a243e26)

Reply:
@vmittal-msft the unit tests uses canned data in the mock_tables. The CLI show/config commands operate on CONFIG_DB and does not involve hardware programming. As such, it should be irrelevant whether or not the pool is HBM or not. Here's a code reference to the mock_tables: https://github.com/sonic-net/sonic-utilities/blob/d1ca905e3b0733170d19abeecad1cfbfd0180698/tests/mock_tables/config_db.json#L1958